ogr/ogrsf_frmts/parquet/ogrparquetdriver.cpp

```diff
@@ -254,8 +255,50 @@ static GDALDataset *OpenFromDatasetFactory(
     const bool bIsVSI = STARTS_WITH(osBasePath.c_str(), "/vsi");
     if (bIsVSI)
     {
-        PARQUET_THROW_NOT_OK(scannerBuilder->FragmentReadahead(2));
-        // scannerBuilder->BatchSize(10);
+        const int nFragmentReadAhead =
```

Comment:
are these changes related to the bugfix?

Reply:
not directly. Various tunables I tried (unsuccessfully) to diagnose things. The bugfix itself is the last commit.